### PR TITLE
LPS-140224 Accessibility in Repeatable fields

### PR DESCRIPTION
**Pull request by rijema**
Related Issue: https://issues.liferay.com/browse/LPS-140224

Click [here](https://issues.liferay.com/browse/LPS-140224) to see the LPS 

In this pull I'm updating the legend of some buttons in our Forms. Let me know if I have to change something.

Thanks!

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rijema, can you please check if these failures are related to your changes? Thanks

**Review by carolmariaabb — CHANGES_REQUESTED** [review]
Hi @rijema,

Some of these failures seem to be related to your changes, like the `FormsTextField#CanBeSetAsRepeatable` and the unit test. To fix the unit test, please run `yarn test -- test/js/FieldBase/ReactFieldBase.es.js -U` in `dynamic-data-mapping-form-field-type` module.

In addition, please see my comments below.

Thanks!

**Comment by rijema** [comment]
Hi @carolmariaabb , thanks for the reply. I just sent a new update to this PR. I didn't know about the title attribute, thanks for the info! I will try to rerun the tests to see if it works now.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 1]
```diff
@@ -282,7 +289,6 @@ function FieldBase({
 							})
 						}
 						small
-						title={Liferay.Language.get('duplicate')}
```
[the remark this answers is not in the document]
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js]: Done

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 2]
```diff
@@ -260,14 +264,17 @@ function FieldBase({
 								})
 							}
 							small
-							title={Liferay.Language.get('remove')}
```
[the remark this answers is not in the document]
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js]: I just reinserted the title field to the FieldBase. Should I just choose just one to keep? I didn't know if I would affect the product if changed the message of this item, so I inserted the requested without changing the previous info.

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rijema, can you please fix the source formatter issues and resend? Thanks

**Pull request closed without merging**